Example/Stripe iOS Example (Custom)/ViewController.m

```diff
@@ -142,7 +142,9 @@ - (IBAction)beginStripeCheckout:(id)sender {
     STPCheckoutOptions *options = [[STPCheckoutOptions alloc] initWithPublishableKey:[Stripe defaultPublishableKey]];
     options.purchaseDescription = @"Cool Shirt";
     options.purchaseAmount = 1000; // this is in cents
+    options.requireBillingAddress = @YES;
     options.logoColor = [UIColor purpleColor];
```

Comment:
I'm guessing that this color isn't the best complement to the Dribble logo. Would you mind substituting the color-detection color in? It's this:

```
hex: "#bd1f5b"
rgb: {r: 188, g: 31, b: 91}
```